frontend/src/store/index.js

```diff
@@ -25,6 +25,144 @@ Vue.use(Vuex)
 export const store = new Vuex.Store({
 
   state: {
+    addIndividualITAsteps: [
+      {
+        step: 1,
+        title:'Exam Type',
+        questions: [
+          {
+            key: 'exam_type_id',
+            text: 'Exam Type ID / Colour',
+            kind:'dropdown',
+            minLength: 0,
+            digit: false,
+          }
+        ]
+      },
+      {
+        step: 2,
+        title:'Exam Info',
+        questions: [
+          {
+            key: 'event_id',
+            text:'Event ID' ,
+            kind: 'input',
+            minLength: 6,
+            digit: true,
+          },
+          {
+            key: 'exam_name',
+            text: 'Exam Name',
+            kind: 'input',
+            minLength: 6,
+            digit: false
+          },
+          {
+            key: 'examinee_name',
+            text: `Exam Writer's Name`,
```

Comment:
Should this be `Examinee Name`?

Reply:
I feel like most people don't even know that examinee is a word or what it means.  I went with exam writer for clarity but I can change it.